### DISCUSSION TITLE
Welcome to xmrig-for-android Discussions!

### DISCUSSION THREAD

**Discussion by garrylachman** [XMRig-for-Android/xmrig-for-android#130] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlvinGy** [comment]
Hi i have a question on how to withdraw the monero i mine

**Comment by amzinglyrwarded** [comment]
I just downloaded the App for Android! 
Linked to SupportXMR and got it running! 

5 accepts so far!
I will be a reliable user if you need test cases.

My biggest question is around security. 
What can you see on your end and what are you tracking??

**Reply by garrylachman** [reply]
The code is open source...

**Reply by Tgood91** [reply]
I can only mine XNO-NANO. XRO, ANA  pretty much micro coins. 

I have 3 free GB of RAM and 8 cpu @ 1%  donation I think my validator is taking ALOT of my coins. Maxes out at 115h/s


And I'm also running
Fresh no used memory
Amd-a53-64bit 1.2 free RAM 4 CORE CPU just a crappy tablet maxes out at 30h/s 
Wondering if how I I find a trustworthy validator ii checked the b.chain and it seems like my set one is sending coins directly from my earning when he gets his, when I withdraw from nanswap., anyone know anything that could help...
All I've taught myself is XNO XRO 
ID LOVE TO LEARN HOW TO MINE XMR WHILE THE PRICE IS LOW.

**Comment by Casper1268** [comment]
Hello, have you gotten this to be stable and run on an android?

**Comment by Tgood91** [comment]
Has anyone used AMD A53-64BIT 2GB

At 1.6GB RAM -30H/S MAX 

I'm just wondering if I could shut some of the configuration switches such as ghostrider.

I'm mining XNO-NANO  
        Have also tried XRO-    RAIBLOCKS        was my wallet. 
I'm not sure if there's anywhere I can search to tell me besides here
If I ask forums anywhere I'll get a leech (,PHISHING)  I'm not even sure if this is an open forum, if not, my apologies.

ANY INFO WOULD BE GREATLY APPRECIATED
THANKS
